README.md

```diff
@@ -2,69 +2,147 @@
 
 # React Native Calendars 🗓️ 📆
 
+## A declarative cross-platform React Native calendar component for iOS and Android.
+
 [![Version](https://img.shields.io/npm/v/react-native-calendars.svg)](https://www.npmjs.com/package/react-native-calendars)
 [![Build Status](https://travis-ci.org/wix/react-native-calendars.svg?branch=master)](https://travis-ci.org/wix/react-native-calendars)
 
-This module includes various customizable **React-Native** calendar components.
+<br>
+
+This module includes information on how to use this customizable **React-Native** calendar component.
```

Comment:
```suggestion
This module includes information on how to use this customizable **React Native** calendar component.
```

Reply:
✅  - Thanks for catching that. Will fix it.